avm/res/app/job/tests/e2e/consumptionPlan/main.test.bicep

```diff
@@ -0,0 +1,66 @@
+targetScope = 'subscription'
+
+metadata name = 'Using a consumption plan'
```

Comment:
General question - is this test case not already covered by the min test which, I assume, defaults to a 'consumption' plan?

Reply:
Yes, no, maybe @ReneHezser ? 😄

Reply:
The default test is not using the consumption plan.
![image](https://github.com/Azure/bicep-registry-modules/assets/5988767/43445281-4334-4853-96c5-e2bb0e1989bc)

The consumption plan test ist using it.
![image](https://github.com/Azure/bicep-registry-modules/assets/5988767/3bb64605-df17-481a-8499-15ceac58ed2d)

I created the additional test to test this.